_posts/newsletters/2023-12-20-newsletter.md

```diff
@@ -0,0 +1,339 @@
+#
+
+<a name="community-calls"></a>
+## Community Calls
+ 
+ 
+ ### January Community Call 
+
+**Call for Presenters: Present Your Favorite Tools or Project!**
+
+We are looking for people who would like to present their favorite tool or project at our next Community Call on Thursday, January 11th, at 12pm ET/9am PT/11am CST. Presentations would be 10-15 minutes long and you can present your own project or a tool you regularly use and think everybody should know about! If you are interested in presenting, please get in touch with Julia Damerow and Abbey Roelofs in the [#communitycalls](https://app.slack.com/client/T8ZT4PJSW/CLBDQMJH5) channel or by messaging them directly.
+
+
+### December Community Call (Past)
+**The December Community Call was on the topic of Fuzzy Requirements**
+
+Have you been in the situation that a researcher you work with is unclear about the requirements for the software you are developing? How do you handle situations in which the requirements are conflicting, the goal of the project is unclear, or the focus is shifting frequently? This was an opportunity to share tools or techniques as well as brainstorm solutions with others who may have dealt with similar situations.
+
+
+### November Community Call (Past)
+
+**The November Community Call was a [discussion with our new Executive Director, Sandra Gesing](https://youtu.be/MzOne9PXa1I) about Envisioning the Next Stage for US-RSE and Synergies with the Center of Excellence for Science Gateways (SGX3)**
+
+
+US-RSE has had a lot of momentum and positive energy from the beginning in 2017. It gained even more in the last couple of months evident in the rising numbers of members growing to over 2000 and the sold out inaugural conference in Chicago. The grant of the Sloan foundation allows for the first time that staff members can complement the volunteer effort: Sandra Gesing as the Executive Director at 50% and a soon-to-be-announced full-time community manager. These roles will take US-RSE to the next level beyond volunteer work. The other 50% of Sandra’s effort go to science gateways aka virtual research environments aka virtual labs, which are research software solutions that allow researchers to focus on their research and not on nitty-gritty details of complex research infrastructures. The community of RSEs and science gateways have quite some overlap. A common goal of both communities is the creation of non-traditional career paths  for RSEs and science gateway creators that are lacking in the academic and national lab world. The community call covered Sandra’s vision for the role of the executive director to achieve financial sustainability for US-RSE, collaboration via the Center of Excellence for Science Gateways (SGX3), and major tasks for the next couple of months.
+
+
+
+
+<a name="events"></a>
+# **Interesting Events and Opportunities**
+
+## Community and Travel Funds program
```

Comment:
I worry that people won't see this because it's "so far down."

Reply:
Want to suggest a change?

Reply:
Made a suggestion that would also include removing the verbiage from here.

Reply:
What about just leaving it? It's not long and never a bad thing to remind people (especially if they skip to there).

Reply:
Agreed. LEAVE IT!